builtins/amp-img.js

```diff
@@ -167,6 +169,17 @@ export class AmpImg extends BaseElement {
     return true;
   }
 
+  /** @override */
+  firstLayoutCompleted() {
+    const placeholder = this.getPlaceholder();
+    if (isExperimentOn(this.win, 'blurry-placeholder') && placeholder && 
+      placeholder.classList.contains('i-amphtml-blur')) {
```

Comment:
I'm a little confused, how is `i-amphtml-blur` set?

Reply:
The devtool/cache generates it.